rllib/agents/ppo/ppo.py

```diff
@@ -72,6 +75,33 @@
 
 
 def choose_policy_optimizer(workers, config):
+    if config["distributed_data_parallel_optimizer"]:
```

Comment:
This config is a bit awkward... we can probably revisit this once we add documentation for this.

Reply:
(i.e., we could add a DD_PPO algorithm that configures these things correctly by default).